s3_management/manage.py

```diff
@@ -14,7 +14,6 @@
 
 
 S3 = boto3.resource('s3')
-CLIENT = boto3.client('s3')
```

Comment:
This was dead code, so drive-by cleanup

Reply:
I used that in https://github.com/pytorch/builder/pull/1433 but maybe S3 will have the same effect?

Reply:
Undoubtedly

Reply:
Oops, I'm a doofus who can't read :facepalm: 

No `S3` won't be sufficient. I guess we'll just leave it in then :stuck_out_tongue:

Reply:
Well, looks like it can go again: https://github.com/pytorch/builder/pull/1433#discussion_r1261885338